sdk/storage/Azure.Storage.Blobs.Cryptography/src/Models/EncryptionData.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Buffers.Text;
+using System.Runtime.Serialization;
+using Metadata = System.Collections.Generic.IDictionary<string, string>;
+
+namespace Azure.Storage.Blobs.Specialized.Cryptography.Models
+{
+    /// <summary>
+    /// Represents the encryption data that is stored on the service.
+    /// </summary>
+    [DataContract]
+    public struct EncryptionData
```

Comment:
Mutable structs that contain other mutable structs are a bad time.  This should be a class.

Reply:
Does this need to be public in v1?

Reply:
It's being serialized. The serialization package needs to be able to see the class.